src/lib/utils/errors.ts

```diff
@@ -1,46 +1,63 @@
 import debounce from 'lodash/debounce';
 import { captureException } from '@sentry/browser';
 import { ScopeContext } from '@sentry/types/types/scope';
+import { useI18n } from 'vue-i18n';
+import { TransactionError } from '@/types/transactions';
+import { TransactionAction } from '@/composables/useTransactions';
+import { UseQueryReturnType } from '@tanstack/vue-query';
 
-type BalancerExceptionAction = 'swap' | 'joinPool' | 'exitPool';
+interface Params {
+  error: Error | unknown;
+  action?: TransactionAction | 'unknown';
+  msgPrefix?: string;
+  context?: Partial<ScopeContext>;
+  query?: UseQueryReturnType<any, any>;
+}
 
 export const captureBalancerException = debounce(
```

Comment:
From a bit of testing, it actually seems to be working as expected. If we take the join pool flow for example. 
1. I threw a test error in the low-level join function in `join-pool.provider.ts`
1. That triggers a `captureBalancerException` with all the relevant context from the join pool form.
1. After that capture, the error is re-thrown and it's caught again in BalActionSteps
1. BalActionSteps also tries `captureBalancerException`, but I think because of the debounce, that capture isn't processed.

This is the behaviour we want, isn't it? The low-level calls with the context are the ones sent to Sentry. i.e. the source error is sent and captures higher up the stack won't be sent to sentry if they have been already.

Reply:
Ahh, I've tried going deeper to the `sendTransaction` call and indeed, we lose the context from the extra params.